scala3doc/src/dotty/dokka/tasty/TypesSupport.scala

```diff
@@ -168,7 +174,10 @@ trait TypesSupport:
             case Seq(rtpe) =>
               text("() => ") :: inner(rtpe)
             case Seq(arg, rtpe) =>
-              inner(arg) ++ texts(" => ") ++ inner(rtpe)
+              val ar = arg match
```

Comment:
We can use more meaningful names for the values
```
val argsList = arg match
  case  byName: ByNameType => texts("(") ++ inner(byName) ++ texts(")")
  case _ => inner(arg)
```

Reply:
Yeah, my bad, fixed